README.md

```diff
@@ -67,7 +65,7 @@ statful.counter('transactions', 1);
 
 > **IMPORTANT:** This configuration uses the default **host** and **port**. You can find these default values in the below table. 
 [More configurations are available in the [Examples](#examples) section].  
-The host and port need to be added within the api Object as shown in the above code.  
+The host and port need to be added as shown in the above code, within the api or udp Object as applicable.  
 Your token value is available in the 'Api Tokens' page of the Website.
```

Comment:
"Your token value is available in the 'Api Tokens' page of the Website." to "Your token value is available in the 'Api Tokens' page on our Statful app account"